### ISSUE TITLE
CI failure in JDK < 8

### ISSUE THREAD

**Issue by davidmoten** [aws/aws-sdk-java#1731]
For JDK < 8 all builds are failing with:
```
[ERROR]   The project com.amazonaws:aws-java-sdk-osgi:1.11.392-SNAPSHOT (/home/travis/build/aws/aws-sdk-java/aws-java-sdk-osgi/pom.xml) has 1 error
[ERROR]     Unresolveable build extension: Plugin org.apache.felix:maven-bundle-plugin:2.5.3 or one of its dependencies could not be resolved: Failed to read artifact descriptor for org.apache.felix:maven-bundle-plugin:jar:2.5.3: Could not transfer artifact org.apache.felix:maven-bundle-plugin:pom:2.5.3 from/to central (https://repo.maven.apache.org/maven2): Received fatal alert: protocol_version -> [Help 2]
```

For example: https://travis-ci.org/aws/aws-sdk-java/jobs/418486828

**Comment by varunnvs92** [comment]
Thanks for reporting the issue. We are aware of it and have a task to fix it this week. We will update the issue when its done.

**Comment by varunnvs92** [comment]
Might be related to 
https://github.com/travis-ci/travis-ci/issues/9763
https://central.sonatype.org/articles/2018/May/04/discontinued-support-for-tlsv11-and-below/?__hstc=31049440.01d70cd8a123efa2827db80259ca61bf.1530207152503.1530231885049.1534984717854.3&__hssc=31049440.1.1534984717854&__hsfp=2725607461

**Comment by zoewangg** [comment]
Fixed in 1.11.394
